### PR TITLE
Swope mce work

### PR DESCRIPTION
**Pull request by swopebe**

### CODE REVIEW

File: multicluster_engine/about/mce_intro.adoc [hunk 1]
```diff
@@ -1,10 +1,8 @@
 [#mce-intro]
 = About the multicluster engine for Kubernetes operator
 
-The {mce} is a software operator that enhances cluster fleet management. The {mce} supports {ocp} and Kubernetes cluster lifecycle management across clouds and data centers. 
-
-See the following documentation:
+The {mce} is a cluster lifecycle operator that provides multicluster lifecycle capabilities for {ocp} and Kubernetes across clouds and data centers. See https://access.redhat.com/articles/6973062[The multicluster engine for Kubernetes operator 2.1 support matrix], as well the following documentation about the {mce}:
```
Comment (oafischer) [inline multicluster_engine/about/mce_intro.adoc]: ...as well as the following... (add missing as)

Or just replace with and.

File: multicluster_engine/about/requirements.adoc [hunk 2]
```diff
@@ -6,34 +6,18 @@ Before you install the {mce}, review the following system configuration requirem
...
 * <<network-configuration,Network configuration>>
 
-*Important:* You must install multicluster engine for Kubernetes on a cluster that does not have Red Hat Advanced Cluster Management for Kubernetes earlier than 2.5 installed. The multicluster engine for Kubernetes cannot co-exist with Red Hat Advanced Cluster Management for Kubernetes on versions earlier than 2.5 because they provide some of the same management components. It is recommended that you install multicluster engine for Kubernetes on a cluster that has never previously installed Red Hat Advanced Cluster Management. If you are using Red Hat Advanced Cluster Management for Kubernetes at version 2.5 or later then multicluster engine for Kubernetes is already installed on the cluster with it.
-
+*Important:* You must install multicluster engine for Kubernetes on a cluster that does not have Red Hat Advanced Cluster Management for Kubernetes earlier than 2.5 installed. If you are using Red Hat Advanced Cluster Management for Kubernetes at version 2.5 or later, then multicluster engine for Kubernetes is already installed on the cluster with it.
```
Comment (oafischer) [inline multicluster_engine/about/requirements.adoc]: Do we want to use {mce} and {product-title} here?

**Review by oafischer — APPROVED** [review]
Left 2 comments, otherwise looks good. Thanks!

File: multicluster_engine/about/requirements.adoc (hunk 2 above)
Comment (swopebe) [reply multicluster_engine/about/requirements.adoc]: yes, though I didn't in the beginning and will prob pay for it in the end if we pull this out.

**Comment by jelkosz** [comment]
I dont think Im the correct person to review this. It looks good to me but I think someone from MCE should give a +1 on it.

File: multicluster_engine/about/mce_intro.adoc [hunk 3]
```diff
@@ -1,9 +1,7 @@
 [#mce-intro]
 = About the multicluster engine for Kubernetes operator
 
-The {mce} is a software operator that enhances cluster fleet management. The {mce} supports {ocp} and Kubernetes cluster lifecycle management across clouds and data centers. 
-
-See the following documentation:
+The {mce} is a cluster lifecycle operator that provides multicluster lifecycle capabilities for {ocp} and Kubernetes across clouds and data centers. See https://access.redhat.com/articles/6973062[The multicluster engine for Kubernetes operator 2.1 support matrix], as well as the following documentation about the {mce}:
```
Comment (swopebe) [inline multicluster_engine/about/mce_intro.adoc]: This is what I was hoping to get a review on @jnpacker

Comment (jnpacker) [reply multicluster_engine/about/mce_intro.adoc]: For MCE I would remove this:
> and Kubernetes across clouds and data centers.

**Comment by jnpacker** [comment]
Viewing the file I saw this:

>  Post-commit hook from GitHub to the multicluster engine for Kubernetes cluster. This setting is only required when you use certain application management functions.

This is not present in MCE.
There is an Outbound connection we are adding in MCE 2.2 that will communicate with Github to pull in OCP release versions (just the meta data that represents the versions) @philipwu08 if you know if you use ssh vs HTTPS for the clone, that would be helpful.

File: multicluster_engine/about/requirements.adoc [hunk 4]
```diff
@@ -6,34 +6,18 @@ Before you install the {mce}, review the following system configuration requirem
...
 
-*Important:* You must install multicluster engine for Kubernetes on a cluster that does not have Red Hat Advanced Cluster Management for Kubernetes earlier than 2.5 installed. The multicluster engine for Kubernetes cannot co-exist with Red Hat Advanced Cluster Management for Kubernetes on versions earlier than 2.5 because they provide some of the same management components. It is recommended that you install multicluster engine for Kubernetes on a cluster that has never previously installed Red Hat Advanced Cluster Management. If you are using Red Hat Advanced Cluster Management for Kubernetes at version 2.5 or later then multicluster engine for Kubernetes is already installed on the cluster with it.
-
+*Important:* You must install {mce} on a cluster that does not have {product-title} earlier than 2.5 installed. If you are using {product-title-short} version 2.5 or later, then multicluster engine for Kubernetes is already installed on the cluster with it.
 
 [#supported-operating-systems-and-platforms]
-== Supported operating systems and platforms for multicluster engine for Kubernetes clusters and managed clusters
+== Supported operating systems and platforms
```
Comment (jnpacker) [inline multicluster_engine/about/requirements.adoc]: I would change this to:
>Supported browsers and platforms

OR

>Supported platforms and browsers

**Pull request merged**